### ISSUE TITLE
Can we delete proud_navbar?

### ISSUE THREAD

**Issue by jlyon** [proudcity/proud#66]

**Comment by jlyon** [comment]
Seems like proud_toolbar is the module being used. Can we rm proud_toolbar from profiles/proud/modules/custom/proud_navbar?

**Comment by aschmoe** [comment]
No, proud_navbar is the front end one... we could just combine I suppose.

**Comment by jlyon** [comment]
yah lets combine w we get a chance

**Comment by jlyon** [comment]
Maybe nvm on this. let's talk abt it

**Comment by aschmoe** [comment]
Already done.  Sorry.
